confero/settings.py

```diff
@@ -10,12 +10,13 @@
 https://docs.djangoproject.com/en/2.1/ref/settings/
 """
 
+# flake8: noqa
```

Comment:
Tells prospector not to lint this file, since it complains about the secret key in here